Travis CI fails

```
CMake Error at CMakeLists.txt:371 (message):

  You must install SDL2 to compile the DDNet client

-- Automatically downloading GTest to be able to run tests

-- Configuring done

-- Generating done

-- Build files have been written to: /Users/travis/build/ddnet/ddnet7/build_debug/googletest-download

Scanning dependencies of target googletest

[ 11%] Creating directories for 'googletest'

[ 22%] Performing download step (git clone) for 'googletest'

Cloning into 'googletest-src'...

Note: checking out 'release-1.8.1'.

You are in 'detached HEAD' state. You can look around, make experimental

changes and commit them, and you can discard any commits you make in this

state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may

do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 2fe3bd99 Merge pull request #1433 from dsacre/fix-clang-warnings

[ 33%] No patch step for 'googletest'

[ 44%] Performing update step for 'googletest'

[ 55%] No configure step for 'googletest'

[ 66%] No build step for 'googletest'

[ 77%] No install step for 'googletest'

[ 88%] No test step for 'googletest'

[100%] Completed 'googletest'

[100%] Built target googletest

-- Looking for WavpackOpenFileInputEx

-- Looking for WavpackOpenFileInputEx - not found

-- Configuring incomplete, errors occurred!

See also "/Users/travis/build/ddnet/ddnet7/build_debug/CMakeFiles/CMakeOutput.log".

See also "/Users/travis/build/ddnet/ddnet7/build_debug/CMakeFiles/CMakeError.log".

The command "cmake -Werror=dev -DDOWNLOAD_GTEST=ON -DDEV=ON .." exited with 1.

0.02s$ make run_tests

make: *** No rule to make target `run_tests'.  Stop.

The command "make run_tests" exited with 2.

0.05s$ cd ../build

The command "cd ../build" exited with 0.

0.01s$ ./DDNet-Server shutdown

/Users/travis/.travis/functions: line 109: ./DDNet-Server: No such file or directory

The command "./DDNet-Server shutdown" exited with 127.
```
https://travis-ci.org/ddnet/ddnet7/builds/622053658

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,6 @@
 os: osx
 script:
+- brew update
 - brew install sdl2
 - mkdir build; cd build
 - cmake -Werror=dev -DDOWNLOAD_GTEST=ON ..

